package.json

```diff
@@ -20,6 +20,8 @@
     "cssrecipes-grid": "^1.0.0",
     "cssrecipes-reset": "^0.5.0",
     "cssrecipes-utils": "^0.6.2",
+    "fontfaceobserver": "^2.1.0",
```

Comment:
We wouldn't need to add these here, unless they're dependencies for our build tool chain.

Reply:
Makes sense. I thought we'd have to use it but we didn't and I forgot to delete.
Thanks Mike!